packages/p2p/src/utils/height_constants.js

```diff
@@ -5,6 +5,7 @@ export const height_constants = Object.freeze({
     core_footer: '37px', // 36px + 1px border.
     filters: '44px',
     filters_margin: '4rem', // 2.4rem + 1.6rem.
+    mobile_content: '37.3rem',
```

Comment:
Do we have to use constant value here? Isn't this problematic on different device screen sizes?